README.md

```diff
@@ -130,6 +132,26 @@ jobs:
       - run: rebar3 ct
 ```
 
+### Basic example (`rebar3` on Windows 2016)
+
+```yaml
+# create this in .github/workflows/ci.yml
+on: push
+
+jobs:
+  test:
+    runs-on: windows-2019
+    name: Erlang/OTP ${{matrix.otp}} / rebar3 ${{matrix.rebar3}}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: '24.0.2'
+          rebar3-version: '3.16.1'
+      - shell: pwsh
+        run: "& rebar3 ct"
```

Comment:
this can also be:

```suggestion
      - run: rebar3 ct
```

(see: https://github.com/wojtekmach/test_erlang/blob/1fe6ff4c8bd491e7911e4207c8a73acb58c04fde/.github/workflows/main.yml#L32 & https://github.com/wojtekmach/test_erlang/runs/2888692249?check_suite_focus=true)

Reply:
Oh, this is so much better. Let me do that.

Reply:
I'll also try this approach in `test.yml`. I didn't know `shell` was a thing. Have not completely read the specification, so am working on a per-need basis 😄